src/test/moment/duration.js

```diff
@@ -464,6 +464,24 @@ test('humanize duration with suffix', function (assert) {
     assert.equal(moment.duration({seconds:  44}).humanize(true),  'in a few seconds', '44 seconds = a few seconds');
     assert.equal(moment.duration({seconds: -44}).humanize(true),  'a few seconds ago', '44 seconds = a few seconds');
     assert.equal(moment.duration({seconds: +44}).humanize(true),  'in a few seconds', '44 seconds = a few seconds');
+    assert.equal(moment.duration({seconds:  44}).humanize({withSuffix: true}),  'in a few seconds', '44 seconds = a few seconds');
+    assert.equal(moment.duration({seconds: -44}).humanize({withSuffix: true}),  'a few seconds ago', '44 seconds = a few seconds');
+    assert.equal(moment.duration({seconds: +44}).humanize({withSuffix: true}),  'in a few seconds', '44 seconds = a few seconds');
+});
+
+test('humanize duration with options', function (assert) {
+    var thresholds = {s: 9};
+    moment.locale('en');
+    assert.equal(
+        moment.duration({seconds: -10}).humanize({thresholds: thresholds}),
+        'a minute',
+        '10 seconds = a minute (with thresholds)'
+    );
+    assert.equal(
+        moment.duration({seconds: 10}).humanize({thresholds: thresholds, withSuffix: true}),
+        'in a minute',
+        '100 seconds = a minute (with thresholds and suffix)'
```

Comment:
`{seconds: 10}` VS "100 seconds`, one of these is wrong.

Reply:
Looks like the message has an extra zero. I'll put up a fix.